src/ec/packages/ec-component-select/select.test.js

```diff
@@ -0,0 +1,102 @@
+import path from 'path';
+import { renderTwigFile } from '@ecl-twig/test-utils';
+
+describe('EC - Select', () => {
+  const template = path.resolve(__dirname, './select.html.twig');
+
+  describe('Default', () => {
+    test('renders correctly', done => {
+      expect.assertions(1);
+
+      renderTwigFile(
+        template,
+        {
```

Comment:
It's not an issue for the size of this component. I'd suggest you either use the spec demo data object as a start and then use `Object.assign` to modify for specific scenarios, adding properties for `invalid_*` and `helper_text` for instance, things which are not given from the reference demo data.